.github/scripts/ensure_actions_will_cancel.py

```diff
@@ -46,11 +46,20 @@ def should_check(filename: Path) -> bool:
             "group": concurrency_key(filename),
             "cancel-in-progress": True,
         }
-        if data.get("concurrency", None) != expected:
+        actual = data.get("concurrency", None)
+        if actual != expected:
             print(
                 f"'concurrency' incorrect or not found in '{filename.relative_to(REPO_ROOT)}'",
                 file=sys.stderr,
             )
+            print(
+                f"expected: {expected}",
+                file=sys.stderr,
+            )
+            print(
+                f"actual:   {actual}",
+                file=sys.stderr,
+            )
```

Comment:
Just curious: wouldn't that make code more readable? Or even perhaps introduce `def eprint(*args): print(*args, file=sys.stderr)` and use it throughout the code
```suggestion
            print(f"expected: {expected}", file=sys.stderr)
            print(f"actual:   {actual}", file=sys.stderr)
```